src/js/components/channels/twilio-channel-content.jsx

```diff
@@ -1,34 +1,133 @@
 import React, { Component } from 'react';
+import { updateTwilioAttributes, resetTwilioAttributes } from '../../services/integrations-service';
 
 import { ReactTelephoneInput } from '../../lib/react-telephone-input';
 
-export class TwilioChannelContent extends Component {
-    state = {
-        numberValid: false
+import { connect } from 'react-redux';
+import isMobile from 'ismobilejs';
+
+export class TwilioChannelContentComponent extends Component {
+
+    linkTwilioNumber = () => {
+        const {state: {formattedNumber}} = this._telInput;
```

Comment:
I'm pretty sure it's against React encapsulation logic to acces a component state directly. I think you should have an `onChange` callback instead and track the value from there. https://stackoverflow.com/questions/27849319/react-is-accessing-the-state-of-children-an-anti-pattern https://facebook.github.io/react/tips/communicate-between-components.html